### ISSUE TITLE
[XACT] FNA seems significantly quieter than XNA

### ISSUE THREAD

**Issue by kiates** [FNA-XNA/FNA#80]
Adjusting the volume by a factor of 2 in the OpenAL wrapper seems to normalize the levels.  I don't know why, but 2x equals 6db, so there may be a 6db error somewhere.  I measured the difference between XACT FACT running the same test application using a white noise sample with a simple iphone based sound pressure meter and they were extremely close after the 2x adjustment.

I could only verify this on windows at the moment, so I'm not sure if there could be platform differences.

**Comment by kiates** [comment]
I'm still trying to find root cause for this.  What I have learned so far is there is a difference in audio levels between positional and non-positional sounds in XACT (non-positional are louder), but in FNA is there is no difference.  In additon, positional sounds in XACT and FNA are approximately equal in my comparisons, so the issue is that XACT non-positional sounds are inherently louder for some reason.

FNA simulates non-positional sounds by staying within the clamped area of the default OpenAL automatic distance attenuation model (Inverse Clamped).  This appears to be working fine, as I temporarily turned of the distance model to compare and the volume levels remained the same.

Further analysis surrounding why XACT non-positional sounds are louder is necessary.

**Comment by flibitijibibo** [comment]
My guess is that this is part of XAudio2 behavior. I really wish I could trace XAudio2 calls, as that would tell us everything we need to know.

**Comment by flibitijibibo** [comment]
Likely relevant to this is the IXACT3Cue interface:

https://github.com/hrydgard/minidx9/blob/master/Include/xact3.h#L939

Apply3D probably calls a bunch of things that are otherwise uncalled, meaning Cues are treated differently without the added mix information.

**Comment by flibitijibibo** [comment]
The latest commits do a lot of scary things to volume storage and calculation, so let's make sure we're at least as "good" as we were before (or possibly better if wizardry occurred).

**Comment by kiates** [comment]
Ok, I'll integrate your new changes and compare/contrast.

**Comment by flibitijibibo** [comment]
Closing as WONTFIX in the existing audio implementation, and WAITING for the FAudio rewrite coming up.

I'm pretty sure this was caused by the sound having different volumes with the default matrix coefficients, followed by X3DAudioCalculate doing a bunch of things alongside whatever XACT volume RPCs you might have. Once this pull request is merged, this should be done unless I'm wrong (test this whenever you can):

https://github.com/flibitijibibo/FACT/pull/8
